iron-list.html

```diff
@@ -729,6 +775,8 @@
 
       // increase the pool of physical items
       this._increasePoolIfNeeded();
+
+      this.fire('iron-list-dom-updated', null, {bubles: false});

```

Comment:
Oh, I forgot to remove this code 😅. I only added it to build basic accessibility in the demo instead of iron-list itself, but the new approach is much simpler and DOM states such as focus are preserved automatically.